README.md

```diff
@@ -49,23 +53,23 @@ func buildImageCache() -> AnyThrowingAsyncCache<URL, UIImage> {
         .mapValue { _, image in
             image
         }
-        .storage((WeakObjectStorage())
-        .coordinated() // Always finish an `async` cache chain with this one. You usually need only one at the end.
+        .cache((WeakObjectCache())
+        .coordinated() // You should usually finish an `async` provider chain with this one.
```

Comment:
Would be good to specify the "why" here so it's more memorable why forgetting to do so will lead to a bad time.

Reply:
Good idea. Reworded.